README.md

```diff
@@ -56,6 +56,12 @@ When you want to upgrade those components — which we advise doing regularly 
 docker-compose pull
 ```
 
+You can also pass the `--pull` argument to `./bin/start`:
+
+```bash
+./bin/start --pull
+```
+
```

Comment:
I suggest replacing the `docker-compose pull` command with this rather than having it as an addition.